plugins/cn0508.c

```diff
@@ -0,0 +1,323 @@
+/**
+ * Copyright (C) 2019 Analog Devices, Inc.
+ *
+ * Licensed under the GPL-2.
+ *
+ **/
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <gtk/gtk.h>
+#include <gtkdatabox.h>
```

Comment:
I don't think you need to include all these headers that all related to gtkdatabox.